build.sbt

```diff
@@ -369,7 +369,7 @@ lazy val docs = project
       "io.opentelemetry" % "opentelemetry-sdk-extension-autoconfigure" % OpenTelemetryVersion
     ),
     mdocVariables := Map(
-      "VERSION" -> version.value,
+      "VERSION" -> tlLatestVersion.value.getOrElse(version.value),
```

Comment:
That way we will show only stable releases, rather than snapshots. 
WDYT?

Reply:
probably a good idea

Reply:
Hmm, sbt-typelevel already injects an appropriate value for `VERSION`, I don't think we need to override it.
https://github.com/typelevel/sbt-typelevel/blob/d2c50f718371c4b9d52f564b8049af5e43334379/site/src/main/scala/org/typelevel/sbt/TypelevelSitePlugin.scala#L117

Reply:
Thanks. I removed the explicit `VERSION`.